src/raven/spec.clj

```diff
@@ -0,0 +1,9 @@
+(ns raven.spec
+  "specifications for the wire JSON structure when talking to sentry."
+  (:refer-clojure :exclude [assert])
+  (:require [clojure.spec.alpha :as s]
+            [expound.alpha      :as expound]))
+
+; We declare the message spec in the raven.client namespace to allow easy
+; reference from there (simply "::payload" when using the spec).
```

Comment:
nitpick: the convention is two columns to start comments.

Reply:
Done.